Automattic/wp-calypso
# Social Logos: update to the latest version, with X logo.

Related:

- https://github.com/Automattic/social-logos/pull/126
- https://github.com/Automattic/jetpack/pull/33134

## Proposed Changes

This updates the Social Logos package in the Calypso, so we can use the new X logo.

## Testing Instructions

> **Note**
> This can be tested in coordination with https://github.com/Automattic/jetpack/pull/33134

* Create a new site and set things up following the instructions in the Jetpack PR.
* In Calypso, go to Tools > Marketing > Sharing Buttons
* The X button should be displayed properly.

<img width="788" alt="image" src="https://github.com/Automattic/wp-calypso/assets/426388/fbb145ed-44fe-4d1f-ba61-4ad5be162166">

## Pre-merge Checklist
